spec/unit/pops/loaders/loader_spec.rb

```diff
@@ -433,7 +412,7 @@ module Loader
               logs = []
               Puppet::Util::Log.with_destination(Puppet::Test::LogCollector.new(logs)) do
                 expect(Loaders.find_loader('c').discover(:task)).to(
-                  contain_exactly(tn(:task, 'environment::envtask'), tn(:task, 'globtask'), tn(:task, 'c::foo')))
+                  contain_exactly(tn(:task, 'c::foo')))
```

Comment:
Why simplify the test?

Reply:
I removed those fixture tasks, since we don't actually support tasks defined at the environment level outside of modules.